platform/conf/conf.go

```diff
@@ -266,11 +282,18 @@ func (c *Conf) copyKeysIgnitionV1(keys []*agent.Key) {
 }
 
 func (c *Conf) copyKeysIgnitionV2(keys []*agent.Key) {
-	var keyObjs []v2types.SSHAuthorizedKey
+	c.ignitionV2.Passwd.Users = append(c.ignitionV2.Passwd.Users, v2types.User{
```

Comment:
This change just restores the code removed by c99fa78bcac8c202b01f5e20ff7a01f2012f9760.  I believe you're correct, but it's not currently an issue with the existing tests.